insights/core/__init__.py

```diff
@@ -241,6 +241,37 @@ def get(self, item, default=None):
         return self.data.get(item, default)
 
 
+class CommandParser(Parser):
+    """
+    This class checks output from the command defined in th spec to be sure line
+    do not contain certain string values and that there is only one line.
+    """
+
+    bad_lines = ["No such file or directory", "Command not found", "command not found"]
+
+    def validate_lines(self, results):
+        if results and len(results) == 1:
```

Comment:
You should add a docstring to this function so that it will have a description in the documentation.  Also `bad_lines` should probably be documented.  You can do this by adding a docstring comment in triple quotes under the declaration of the attribute.